dashboard/app/views/levels/editors/fields/_aichat_settings.html.haml

```diff
@@ -5,5 +5,6 @@
   #aichat-settings-editor
 
 - content_for :body_scripts do
+  %script{src: webpack_asset_path("js/#{js_locale}/aichat_locale.js")}
```

Comment:
Nit: I wonder if we should add a comment here since we don't normally load locale for level edit fields/editors.